lib/id3c/api/datastore.py

```diff
@@ -218,12 +218,12 @@ def fetch_identifier_sets(session: DatabaseSession) -> Any:
     """
     Fetch all identifier sets from the backing database using *session*.
 
-    Returns a list of named tuples with ``name`` and ``description``
+    Returns a list of named tuples with ``name``, ``description``, and ``use``
     attributes.
     """
     with session, session.cursor() as cursor:
         cursor.execute("""
-            select name, description
+            select name, description, use
```

Comment:
Would we ever want use to be a list of values, if an identifier set had multiple `use`s?

Reply:
Good question, I'm not really sure. Would this be the case for `tiny swabs` set?  I believe all of the other sets have a single use so I'm not sure if that's just an outlier or not. There's a related slack thread in the software channel from early July about the use type for each existing set.

Reply:
It can get a little murky. For tiny swabs, they are both collection ids and aliquot ids - but I'm not sure we'd want to bring concept of aliquot ids into our data model (or leave that to the LIMS). 
From the [slack thread](https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1625586692016000) it does seem like the existing cases of a identifier set mapping to multiple uses are due to data errors. 

Perhaps let's keep this enforced as a single value and update later if required.